src/components/widgets/NumberWidget.js

```diff
@@ -38,9 +38,6 @@ export default class NumberWidget extends React.Component {
     this.distance = 0;
     this.onMouseDownValue = 0;
     this.prevPointer = [0, 0];
-
-    this.setValue(this.props.value);
```

Comment:
setValue here was a noop, because we have `if (value === this.state.value) return;` in setValue, and this.state.value is already this.props.value from constructor.